### ISSUE TITLE
Solar Neutron Activator Won't Fill With Nuclear Waste

### ISSUE THREAD

**Issue by ACLuckey** [mekanism/Mekanism#7086]
*Please use the search functionality before reporting an issue. Also take a look at the closed issues!*

#### Issue description:
After completing my fission reactor, I have set up pressurized tubes to pull from the bottom of my waste barrels and run them to a solar neutron activator to produce polonium, but it simply refuses to fill with nuclear waste. The tubes running from the barrels are visibly filled with waste. I am indeed inputting the waste through the bottom as I have read everywhere is necessary, the solar neutron activator has a clear and open view to the sky, and I have a pressurized tube running from the front (output) to my PRC, which is correctly set up to produce polonium pellets, but it just won't accept the waste, and both gauges remain empty. I have tried recreating my entire setup in creative and am running into the same issue so I know it isn't just a problem with my world. I have scoured the forums looking for a solution but no one seems to be having the same problem. Does anyone have any idea what the issue is? 

#### Steps to reproduce:
1.
2.
3.

#### Version (make sure you are on the latest version before reporting):
**Forge:** MC-1.16.5  Forge-36.0.55
**Mekanism:** Mekanism V10
![2021-04-05_15 17 22](https://user-images.githubusercontent.com/81996700/113633672-33f9de80-9622-11eb-8191-db2151ba208c.png)
![2021-04-05_15 17 43](https://user-images.githubusercontent.com/81996700/113633688-3b20ec80-9622-11eb-8457-59bbeda64114.png)

**Other relevant version:** 

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)

[[gist](https://gist.github.com/) / [pastebin](https://pastebin.com/) / etc link here. Please make sure that it isn't set to expire.]

**Comment by pupnewfster** [comment]
Are you sure that is waste the coloration is a bit odd especially given it should be the same color before and after being sent through waste barrels? Try right clicking the tubes with a network configurator to double check what is stored in the various tubes.

**Comment by ACLuckey** [comment]
> Are you sure that is waste the coloration is a bit odd especially given it should be the same color before and after being sent through waste barrels? Try right clicking the tubes with a network configurator to double check what is stored in the various tubes.

Hey! Thanks for the suggestion. I just went and checked both sides with the network reader, and it shows that both are indeed filled with nuclear waste. I did notice the different colors, it is a bit odd, but I am using a shader which is likely to blame for that. 
One thing I noticed was when I clicked on the waste barrels with the network reader, it said "Connected sides: [Up]" 
I would assume that it should say "[Down}" as well? But at the same time, there is waste in the output tubes so I'm not sure if that's even part of the problem.

**Comment by ACLuckey** [comment]
> Are you sure that is waste the coloration is a bit odd especially given it should be the same color before and after being sent through waste barrels? Try right clicking the tubes with a network configurator to double check what is stored in the various tubes.

Thanks for the reply! I checked both sides with the network reader and, as you suspected, the output tubes somehow filled with a different gas (fissile fuel). I'm clueless as to how this happened as I connected the tube from the barrels of waste directly to the activator, both of which are nowhere near any fissile fuel machinery or tubing. The same thing happened in my survival world which is also strange (Possible bug?). Nevertheless, I was able to successfully produce polonium in creative! Now to go fix my survival setup.... Thanks again!
